packages/quantic/package.json

```diff
@@ -11,7 +11,7 @@
     "url": "git+https://github.com/coveo/ui-kit"
   },
   "scripts": {
-    "test": "npm run test:lint && npm run test:typings && npm run test:unit",
+    "test": "npm run test:lint && npm run test:typings && npm run doc:generate && npm run test:unit",
```

Comment:
I don't think we should add the doc generation to the test script. It makes more sense to me to have it as a separate step. In addition, the CI would now generate the documentation twice.

Reply:
The idea behind this is to catch some jsdoc errors automatically instead fo relying solely on PR reviewers Sort of like how we test the typing at the same time. It takes 2 seconds to generate it so IMO generating it twice a few times a month when we actually create new package versions is worth it. I could see about adding it as a separate step in the jenkins job perhaps. But given this is just a Quantic thing, a separate step that appears on every build is maybe a little much.

Reply:
The only advantage I could see is that a developer does not have to execute `npm run doc:generate`. Personally I maintain it does not belong to the `test`script. In the CI, the `doc:generate` script is called before `test` so it wouldn't catch the error sooner.

Reply:
🤦‍♂️ i didn't realize it was already running in the CI. That was the whole point of adding it here. You're 100% correct then. I'll remove it.